public/static/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -0,0 +1,96 @@
+# Setup a Google Drive DVC Remote
+
+Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
+[remote storage](/doc/command-reference/remote).
+
+## Create and Configure a **Google Project**
+
+Since Google Drive has tight API usage quotas, one-time creation and
+configuration of a _Google Project_ is required. The mandatory
+[OAuth](https://developers.google.com/identity/protocols/OAuth2) credentials,
+**client ID** and **client secret**, can be generated only once. Please share
```

Comment:
I added this link to Google OAuth page. I'm just wondering, could we replace part of this doc with a guide from Google instead of having every precise steps? For example https://developers.google.com/drive/api/v3/about-auth

Reply:
@jorgeorpinel referencing Google OAuth page looks to me reasonable. But, unfortunately, both mentioned links guide users through details of final users authorization process instead of exact steps in UI of how to generate **client ID** and **client secret**.

Reply:
OK, thanks.